src/op_analytics/datasources/l2beat/tvsbreakdown.py

```diff
@@ -206,8 +206,8 @@ def create_row(esc: dict[str, Any] | None = None) -> dict[str, Any]:
                     "chain_name": chain_name,
                     "chain_id": chain_id,
                     "amount": asset["amount"],
-                    "usd_value": float(asset["usdValue"]) if "usdValue" in asset else None,
-                    "usd_price": float(asset["usdPrice"]) if "usdPrice" in asset else None,
+                    "usd_value": float(asset["valueForProject"]) if "valueForProject" in asset else None,
+                    "usd_price": float(asset["valueForProject"]) / float(asset["amount"]) if "valueForProject" in asset and "amount" in asset and asset["amount"] > 0 else None,
```

Comment:
i think we should still keep this as `value` here given that `valueForProject` is adjusted for double counting, but the `value` is unadjusted value based on amount * price. 

so there could be the case when valueforproject is 0 due to double counting removal, while the value is a number > 0. 

also agreed we can leave out value at this point!